Further tweaks for performance testing

TobySaundersGDS opened the pull request:
let ECS service bring in *5 number of targets
keep targets registered with load balancer for longer set separate scale in and out policies

## Proposed changes
Set scaling policy to handle scale out events faster and scale in more slowly

### What changed
Different policies for each type of scaling

### Why did it change
Improve the FE handling of spiky traffic

### Issue tracking
PYIC-1811

lwebb-iig commented on deploy/template.yaml:
```diff
@@ -440,6 +435,26 @@ Resources:
+    DependsOn: CoreFrontAutoScalingTarget
+    Type: AWS::ApplicationAutoScaling::ScalingPolicy
+    Properties:
+      PolicyName: CoreFrontStepScalingPolicy
```
Should we update this name property on both of the resources ie `CoreFrontStepScaleInPolicy`/`CoreFrontStepScaleOutPolicy` respectively?

TobySaundersGDS replied:
yes - will do so

TobySaundersGDS replied:
Gave the policies different names

TobySaundersGDS committed:
Further tweaks for performance testing

let ECS service bring in *5 number of targets
keep targets registered with load balancer for longer
set separate scale in and out policies

TobySaundersGDS committed:
Merge branch 'main' into PYIC-2811-create-step-scaling-policy-and-apply-in-core-front
